src/se/vidstige/jadb/JadbConnection.java

```diff
@@ -25,6 +25,20 @@ public Transport createTransport() throws IOException {
         return new Transport(new Socket(host, port));
     }
 
+    public void connect(String ip, int port) throws IOException, JadbException {
+        Transport main = this.createTransport();
+        main.send("host:connect:" + ip + ":" + port);
```

Comment:
Can you link to the documentation where you found this part of the protocol? E.g. the "host:connect" string? Thanks in advance.